### PR TITLE
Bug 1304489, clarified how a PVC determines what PV to select

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1304489

Follow-up to https://github.com/openshift/openshift-docs/pull/1683.

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@markturansky @TheDiemer PTAL. Thanks!

File: architecture/additional_concepts/storage.adoc [hunk 1]
```diff
@@ -192,11 +192,23 @@ For example, NFS can support multiple read/write clients, but a specific NFS PV
...
-For example, if a claim requests RWO, but the only volume available was an NFS PV
-(RWO+ROX+RWX), the claim would match NFS because it supports RWO. Direct matches
-are always attempted first.
+Claims are matched to Volumes with similar access modes. The only two matching
+criteria are access modes and size. A claim's access modes represent a request.
+Therefore, the user may be granted more, but never less. For example, if a claim
+requests RWO, but the only volume available was an NFS PV (RWO+ROX+RWX), the
+claim would match NFS because it supports RWO.
+
+Direct matches are always attempted first. The modes must match or contain more
```
Comment (markturansky) [inline architecture/additional_concepts/storage.adoc]: Just for clarity, call out who owns the modes we're looking at?

  "the volume's modes must match or contain..."

File: architecture/additional_concepts/storage.adoc [hunk 2]
```diff
@@ -192,11 +192,23 @@ For example, NFS can support multiple read/write clients, but a specific NFS PV
...
+modes than you requested. The size must be greater than or equal to what is
+expected. If two types of volumes (NFS and iSCSI, for example) both have the
+same set of access modes, then either of them will match a claim with those
+modes. There is no ordering between types of volumes.
+
+If you have two types of volumes with the same access modes, then here is no
+way to choose one type over another. All volumes with the same modes are
```
Comment (markturansky) [inline architecture/additional_concepts/storage.adoc]: This sentence and the last sentence of the previous paragraph kind of say the same thing.

Perhaps extend the previous sentence with "There is no ordering between types of volumes and no way to choose one type over another." and then the next paragraph starting with "All volumes with the same modes..."?

**Comment by markturansky** [comment]
I offered some wording suggestions but the content as written is correct.

**Comment by markturansky** [comment]
LGTM!

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../architecture/additional_concepts/storage.html[Persistent Storage]
|In the link:../architecture/additional_concepts/storage.html#pvc-access-modes[Access Modes] section, clarified how a PVC determines what PV to select.
%

**Pull request merged**